engines/hdb/sound.h

```diff
@@ -1457,21 +1457,38 @@ struct SoundCache {
 };
 
 struct Song {
-	bool playing;
-	SoundType song;
+
```

Comment:
Shouldn't it be class then?

Reply:
Doesn't really matter, either keyword declares a class. `struct` simply means I don't need a `public:` before the member functions.